Directory.Build.props

```diff
@@ -30,8 +30,8 @@
   <!-- Package Validation -->
   <PropertyGroup>
     <GenerateCompatibilitySuppressionFile>false</GenerateCompatibilitySuppressionFile>
-    <EnablePackageValidation>true</EnablePackageValidation>
-    <PackageValidationBaselineVersion>12.0.0</PackageValidationBaselineVersion>
+    <EnablePackageValidation>false</EnablePackageValidation>
```

Comment:
We might want to add the following comment, so we don't forget to re-enable package validation again:
```suggestion
    <!-- TODO: Enable package validation when 13.0.0 is released to NuGet -->
    <EnablePackageValidation>false</EnablePackageValidation>
```

Reply:
I added a step in our release step document to ensure we remember